### DISCUSSION TITLE
Do I need to use fstrim after installing SL?

### DISCUSSION THREAD

**Discussion by birds-swim** [SpiralLinux/SpiralLinux-project#469] [Q&A] [answered]
Hello. 

I'm using an M.2 (1 TB). I noticed in the Btrfs documentation that the options available for SSD/M.2 users are `ssd` and `discard` (I picked the async version).

But it's not entirely clear to me if I need to use fstrim (maybe once or as a background service) with Btrfs.

Any thoughts for this?

**Comment by geckolinux** [comment, marked as answer]
Hi there, you should be good out-of-the-box. Just make sure this timer is set:
`sudo systemctl status fstrim.timer`

**Reply by birds-swim** [reply]
Thanks Geck!
